recipes/libx265/all/conanfile.py

```diff
@@ -49,6 +49,9 @@ def configure(self):
     def source(self):
         tools.get(**self.conan_data["sources"][self.version])
         os.rename("x265_{}".format(self.version), self._source_subfolder)
+        if self.version in self.conan_data["patches"]:
+            for patch in self.conan_data["patches"][self.version]:
+                tools.patch(**patch)
```

Comment:
```suggestion
        for patch in self.conan_data.get("patches", {}).get(self.version, []):
            tools.patch(**patch)
```
And also please move this to the build method so the sources are kept pristine.
When you apply the patch in the `source` method, there would be no point in storing the patches with the sources.

Reply:
Done. Thanks.

Reply:
@madebr Sorry I was not familiar with GitHub's "Commit suggestion" feature. I didn't mean to not give you credit for the change.

Reply:
Don't worry, my patch was incomplete as I didn't add it to the build method